administrator/components/com_tags/src/Helper/AssociationsHelper.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_tags
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Tags\Administrator\Helper;
+
+use Joomla\CMS\Association\AssociationExtensionHelper;
+use Joomla\CMS\Language\Associations;
+use Joomla\CMS\Table\Table;
+use Joomla\Component\Tags\Site\Helper\AssociationHelper;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Content associations helper.
```

Comment:
Should this say 

```suggestion
 * Tags associations helper.
```

If so then other comments should be updated as well